README.md

```diff
@@ -53,6 +53,16 @@ src/MolSim ../input/eingabe-sonne.txt -e 1000
 ```
 src/MolSim ../input/clusters.txt -c -s 1 -d 0.0002 -e 5
 ```
+#### Assignment 3 simulation
+Cuboid collision:
+```
+src/MolSim ../input/assign3.xml -x -s 3
+```
+Falling drop:
+```
+src/MolSim ../input/falling_drop.xml -x -s 3
```

Comment:
I thought of listing the meaning of the options, but thought this would get too long for a help message. 
As specified in the help message, one can read all the details in the man page. 
I think this suffices for now :)